DependencyInjection/Configuration.php

```diff
@@ -50,6 +50,9 @@ public function getConfigTreeBuilder()
                             ->scalarNode('path')
                                 ->defaultValue('%kernel.root_dir%/../web/sitemap.xml')
                             ->end()
+                            ->scalarNode('exclude_urls')
+                                ->defaultValue('')
```

Comment:
As per `protected $excludeUrlsPattern = null;` the default value should be `null`